tools/openms/OpenSwathWorkflow.xml

```diff
@@ -10,14 +10,25 @@
   <expand macro="references"/>
   <expand macro="stdio"/>
   <expand macro="requirements"/>
-  <command detect_errors="aggressive"><![CDATA[OpenSwathWorkflow
+  <command detect_errors="exit_code"><![CDATA[
```

Comment:
Why change from aggressive to exit_code?

Reply:
Because OpenMS is writing warnings or things like that with "Error:" ... its wired, but users ensured me that despite the fact the dataset is red the result is expected.

Reply:
Good to know. Then I will adapt accordingly in the CTDconverter. 

Would be good to have an example to report upstream.